src/console/controllers/MigrateController.php

```diff
@@ -202,7 +202,6 @@ public function beforeAction($action): bool
             }
 
             $this->migrationPath = $this->getMigrator()->migrationPath;
-            FileHelper::createDirectory($this->migrationPath);
```

Comment:
I'm pretty sure this is superfluous, as later we call `parent::beforeAction`, which already tries to create this dir.